### PR TITLE
Addede solution for hw-3.3 palindrome

### PR DESCRIPTION
**Pull request by nevskyy**

### CODE REVIEW

File: src/main/java/core/basesyntax/Palindrome.java [hunk 1]
```diff
@@ -21,6 +21,12 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
+        String clean = text.replaceAll("[\\W]", "").toUpperCase();
+        String reverseString = new StringBuffer(clean).reverse().toString();
+        System.out.println(reverseString);
+        if (clean.equals(reverseString)) {
+            return true;
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Palindrome.java]: what does equals() return?

File: src/main/java/core/basesyntax/Palindrome.java [hunk 2]
```diff
@@ -21,6 +21,12 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
+        String clean = text.replaceAll("[\\W]", "").toUpperCase();
+        String reverseString = new StringBuffer(clean).reverse().toString();
+        System.out.println(reverseString);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Palindrome.java]: remove sout, you can use it to test yourself but don't push it to github)

File: src/main/java/core/basesyntax/Palindrome.java (hunk 1 above)
Comment (nevskyy) [reply src/main/java/core/basesyntax/Palindrome.java]: returns true or false, understood what you mean, resolve soon and send you back

File: src/main/java/core/basesyntax/Palindrome.java (hunk 2 above)
Comment (nevskyy) [reply src/main/java/core/basesyntax/Palindrome.java]: Good morning, well noted, will reply in the nearest. Thanks

**Comment by nevskyy** [comment]
Done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**